endoreg_db/urls/__init__.py

```
@@ -0,0 +1,728 @@
+        # ),
+        # path(
+        #     'patient-finding-morphologies/',
+        #     create_patient_finding_morphology,
+        #     name='create_patient_finding_morphology'
+        # ),
+
+        # ---------------------------------------------------------------------------------------
+
+        #this is for, to test the timeline
+        #need to delete this url and also endoreg_db_production/endoreg_db/views/views_for_timeline.py and endoreg_db_production/endoreg_db/templates/timeline.html
+        path('video/<int:video_id>/timeline/', video_timeline_view, name='video_timeline'),
```

Ruff (0.12.2)

Undefined name `video_timeline_view`

(F821)